.github/workflows/codeforafrica-deploy-prod.yml

```
@@ -74,12 +76,15 @@ jobs:
         uses: docker/build-push-action@v3
         with:
           build-args: |
-            APP=${{ env.APP }}
-            GHOST_API_KEY=${{ secrets.GHOST_API_KEY }}
-            GOOGLE_API_KEY=${{ secrets.GOOGLE_API_KEY }}
+            MONGODB_URL=${{ secrets.CODEFORAFRICA_MONGO_URL }}/${{ env.APP_NAME }}
+            NEXT_PUBLIC_APP_URL=${{ env.NEXT_PUBLIC_APP_URL }}
+            PAYLOAD_SECRET=${{ secrets.CODEFORAFRICA_PAYLOAD_SECRET_KEY }}
+            NEXT_PUBLIC_APP_LOGO_URL=${{ secrets.NEXT_PUBLIC_APP_CFA_LOGO_URL }}
```

```suggestion
            NEXT_PUBLIC_APP_LOGO_URL=${{ secrets.NEXT_PUBLIC_CODEFORAFRICA_APP_LOGO_URL }}
```
Again, lets pick one and stick with it; if its `_CODEFORAFRICA_`, lets use that everywhere, and if it's `_CFA_` lets change everything to use that as well.